# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../chkrootkit/make_install.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2005-02-21 20:13:55.000000000 -0300
+++ ./Makefile	2005-04-27 09:30:10.412337256 -0400
@@ -7,6 +7,8 @@
 CFLAGS	 = -DHAVE_LASTLOG_H
 STATIC   = -static
 
+SBINDIR  = /usr/sbin
+
 ###
 ### Solaris 2.x
 ###
@@ -68,3 +70,8 @@
 
 clean:
 	rm -f ${OBJS} core chklastlog chkwtmp ifpromisc chkproc chkdirs check_wtmpx strings-static chkutmp
+
+install: sense
+	cp -vf chkrootkit $(SBINDIR)
+	cp -vf chklastlog chkwtmp ifpromisc chkproc chkdirs check_wtmpx strings-static chkutmp $(SBINDIR)
+	
